### DISCUSSION TITLE
ST-630-INT Z-Wave

### DISCUSSION THREAD

**Discussion by T365** [zwave-js/zwave-js#2584] [Ideas]
ST-630-INT Z-Wave: Fire Angle Smoke detector
Will this device supported in the future ?, Z-wave JS is awesome..

**Comment by AlCalzone** [comment]
Maybe, but we need information for that. Please open an issue with the "missing configuration file" template.
Or even better, provide the file yourself with a PR. We cannot research and add every device ourselves, but we'll help you with the PR.

**Reply by T365** [reply]
Thank you, I will do that, but can you tell me what a PR is ?

**Reply by AlCalzone** [reply]
A Pull Request - that's how you get changes into someone's Open Source repository.
==> https://guides.github.com/activities/hello-world/
